### PR TITLE
Add missing projection property tables

### PR DESCRIPTION
**Pull request by Sedictious**
Hello, I added all the missing property tables to the projections' docs. I tried to ensure that everything is correct but some mistakes could've slipped in so please tell me if you spot any errors. Hope you find this addition useful!

### CODE REVIEW

File: docs/source/operations/projections/lask.rst [hunk 1]
```diff
@@ -4,6 +4,23 @@
 Laskowski
 ********************************************************************************
 
++---------------------+----------------------------------------------------------+
+| **Classification**  | Miscellaneous                                            |
++---------------------+----------------------------------------------------------+
+| **Available forms** | Forwar spherical projection                              |
```
Comment (kbevers) [inline docs/source/operations/projections/lask.rst]:
```suggestion
| **Available forms** | Forward spherical projection                              |
```

**Review by kbevers — COMMENTED** [review]
This is awesome! Thank you very much for taking the time to do this.

I found a slight typo, see the inlined comment, otherwise I'd say this is good to go

File: docs/source/operations/projections/lask.rst (hunk 1 above)
Comment (Sedictious) [reply docs/source/operations/projections/lask.rst]: Fixed :)

File: docs/source/operations/projections/lask.rst [hunk 2]
```diff
@@ -4,6 +4,23 @@
 Laskowski
 ********************************************************************************
 
++---------------------+----------------------------------------------------------+
+| **Classification**  | Miscellaneous                                            |
++---------------------+----------------------------------------------------------+
+| **Available forms** | Forward spherical projection                              |
```
Comment (rouault) [inline docs/source/operations/projections/lask.rst]:
```suggestion
| **Available forms** | Forward spherical projection                             |
```

**Pull request merged**